src/main.rs

```diff
@@ -184,7 +186,11 @@ fn main() -> Result<(), Box<dyn Error>> {
 fn usage(options: &getopts::Options) {
     println!(
         "Parses the output of `git log`.\n\n{}",
-        options.usage("Usage: commit-analyzer <FILE> [OPTIONS]")
+        options.usage(&format!(
+            "Usage: {} {}",
+            settings::APP_NAME,
+            settings::APP_TOOL_TIP
```

Comment:
Before I submitted the first changes to this project, the source code was built upon something similar:  the application's name as a parameter to be passed to the usage function.  And contained a typo.

The idea is to introduce something like a Single Point Of Truth for the descriptions and / or other magic numbers in order to make changes to them easier since there is obviously only their definition as place to change left.

Reply:
But these constants are only used in one place. There was already a single point of truth. And these two constants are also not expected to change often (or at all probably).

Reply:
In summary:

* move constants directly back to `usage`
* remove modules `application` and `version`

Right? :wrench:

Reply:
Yes 👍

Reply:
:wrench: :heavy_check_mark: